hr_webcam/__init__.py

```diff
@@ -1,12 +1,12 @@
-# -*- coding:utf-8 -*-
+#-*- coding:utf-8 -*-
 #
 #
 #    Copyright (C) 2013 Michael Telahun Makonnen <mmakonnen@gmail.com>.
 #    All Rights Reserved.
 #
 #    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
```

Comment:
Please revert this change

    ./hr_webcam/__init__.py:8:80: E501 line too long (80 > 79 characters)